packages/openapi-react-query/README.md

```diff
@@ -0,0 +1,67 @@
+# openapi-react-query
+
+openapi-react-query is a type-safe tiny wrapper (1 kb) around [@tanstack/react-query](https://tanstack.com/query/latest/docs/framework/react/overview) to work with OpenAPI schema.
+
+It works by using [openapi-fetch](../openapi-fetch) and [openapi-typescript](../openapi-typescript) so you get all the following features:
+
+- ✅ No typos in URLs or params.
+- ✅ All parameters, request bodies, and responses are type-checked and 100% match your schema
+- ✅ No manual typing of your API
+- ✅ Eliminates `any` types that hide bugs
+- ✅ Also eliminates `as` type overrides that can also hide bugs
+
+## Setup
+
+Install this library along with [openapi-fetch](../openapi-fetch) and [openapi-typescript](../openapi-typescript):
+
+```bash
+npm i openapi-react-query openapi-fetch
+npm i -D openapi-typescript typescript
+```
+
+Next, generate TypeScript types from your OpenAPI schema using openapi-typescript:
+
+```bash
+npx openapi-typescript ./path/to/api/v1.yaml -o ./src/lib/api/v1.d.ts
+```
+
+> ⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will err on invalid schemas.
+
+## Usage
+
+Once your types has been generated from your schema, you can create a [fetch client](../openapi-fetch), a react-query client and start querying your API.
```

Comment:
Ignorable suggestion: in the docs, absolute URLs are generally-easier to work with because they work even if this page gets moved or content gets copied to another location. But fortunately Vitepress checks for these (and errs), so it doesn’t really matter too much either way (in other platforms without robust URL checks though it can result in broken links which is where the rule of thumb comes from)

Reply:
I get it for the Vitepress part, I'll do the changes. But is it actually good to use absolute URLs for repository markdown files? If you are reading the doc for a specific version, and navigate using the links, you don't really want to be redirected to the main

Reply:
Oh excuse me! You’re right; this is fine for the README.md files. I got my files mixed up.